components/esp_wifi_remote/CMakeLists.txt

```diff
@@ -1,15 +1,31 @@
 if(NOT CONFIG_ESP_WIFI_ENABLED)
-    set(src_wifi_is_remote esp_wifi_remote.c esp_wifi_with_remote.c)
+    set(src_wifi_is_remote esp_wifi_remote.c esp_wifi_with_remote.c esp_wifi_remote_net.c)
 endif()
 
+#if(CONFIG_ESP_WIFI_REMOTE_LIBRARY_HOSTED)
+#    set(src_wifi_uses_esp_hosted wifi_remote_init.c wifi_remote_rpc.c)
+#endif()
+if(NOT CONFIG_ESP_WIFI_REMOTE_LIBRARY_EPPP)
+    set(src_wifi_remote_weak esp_wifi_remote_weak.c)
+endif()
+
+#if(NOT SLAVE_BUILD)
+#    set(wifi_remote_host_srcs  esp_wifi_remote_net.c)
+#endif()
+
 idf_component_register(INCLUDE_DIRS include
-                    SRCS  ${src_wifi_is_remote}
-                          esp_wifi_remote_net.c
-                          esp_wifi_remote_weak.c
+                    SRCS ${wifi_remote_host_srcs}
+                        ${src_wifi_remote_weak}
+                        eppp/wifi_remote_rpc_defs.c
+#                        eppp/wifi_remote_impl.cpp
```

Comment:
Is it kept for future update? Also this file contains the #if 0 #endif 
Suggest to remove if not needed now.

Reply:
already fixed in the current version